From: win7-support-bot <win7@cromite.org>
Date: Sun, 26 Jul 2026 00:00:00 +0000
Subject: WIN7: tools-metrics / com

Files:
 tools/metrics/histograms/metadata/enterprise/histograms.xml

License: GPL-2.0-or-later - https://spdx.org/licenses/GPL-2.0-or-later.html
---
diff --git a/tools/metrics/histograms/metadata/enterprise/histograms.xml b/tools/metrics/histograms/metadata/enterprise/histograms.xml
index b8e0b7b95f9e1..fae60ef9ee1f5 100644
--- a/tools/metrics/histograms/metadata/enterprise/histograms.xml
+++ b/tools/metrics/histograms/metadata/enterprise/histograms.xml
@@ -2644,6 +2644,7 @@ chromium-metrics-reviews@google.com.
     Captures the response parsing error rate whenever collecting {Signal}.
   </summary>
   <token key="Signal">
+    <variant name="WMI.AntiVirus" summary="AntiVirus info from WMI"/>
     <variant name="WMI.Hotfixes" summary="Hotfixes info from WMI"/>
     <variant name="WSC.AntiVirus" summary="AntiVirus info from WSC"/>
   </token>

